supertokens_python/recipe/emailverification/syncio/__init__.py

```diff
@@ -39,13 +41,27 @@ def verify_email_using_token(
 
 
 def is_email_verified(
-    user_id: str, email: str, user_context: Union[None, Dict[str, Any]] = None
+    user_id: str,
+    email: Optional[str] = None,
+    user_context: Union[None, Dict[str, Any]] = None,
 ):
     from supertokens_python.recipe.emailverification.asyncio import is_email_verified
 
     return sync(is_email_verified(user_id, email, user_context))
 
 
+def revoke_email_verification_token(
```

Comment:
rename this to `def revoke_email_verification_tokens(`, and remove the one below from this file.

Reply:
done